go.sum

```diff
@@ -881,12 +882,11 @@ k8s.io/apimachinery v0.23.3/go.mod h1:BEuFMMBaIbcOqVIJqNZJXGFTP4W6AycEpb5+m/97hr
 k8s.io/client-go v0.23.3 h1:23QYUmCQ/W6hW78xIwm3XqZrrKZM+LWDqW2zfo+szJs=
 k8s.io/client-go v0.23.3/go.mod h1:47oMd+YvAOqZM7pcQ6neJtBiFH7alOyfunYN48VsmwE=
 k8s.io/gengo v0.0.0-20210813121822-485abfe95c7c/go.mod h1:FiNAH4ZV3gBg2Kwh89tzAEV2be7d5xI0vBa/VySYy3E=
-k8s.io/klog v0.3.1 h1:RVgyDHY/kFKtLqh67NvEWIgkMneNoIrdkN0CxDSQc68=
```

Comment:
this is from 2019! 😱 https://github.com/kubernetes/klog/releases/tag/v0.3.1

Reply:
Seems unused, anyway.  Everything uses `/v2` variant.
```
$ go mod graph | grep k8s.io/klog
github.com/mailgun/gubernator/v2 k8s.io/klog@v0.3.1
github.com/mailgun/gubernator/v2 k8s.io/klog/v2@v2.30.0
k8s.io/apimachinery@v0.23.3 k8s.io/klog/v2@v2.30.0
k8s.io/client-go@v0.23.3 k8s.io/klog/v2@v2.30.0
k8s.io/klog/v2@v2.30.0 github.com/go-logr/logr@v1.2.0
k8s.io/kube-openapi@v0.0.0-20211115234752-e816edb12b65 k8s.io/klog/v2@v2.2.0
k8s.io/utils@v0.0.0-20211116205334-6203023598ed k8s.io/klog/v2@v2.0.0
k8s.io/gengo@v0.0.0-20210813121822-485abfe95c7c k8s.io/klog/v2@v2.2.0
k8s.io/klog/v2@v2.2.0 github.com/go-logr/logr@v0.2.0
k8s.io/utils@v0.0.0-20210802155522-efc7438f0176 k8s.io/klog/v2@v2.0.0
k8s.io/klog/v2@v2.0.0 github.com/go-logr/logr@v0.1.0
```